crates/shared/src/sources/uniswap_v3/graph_api.rs

```diff
@@ -323,13 +317,16 @@ mod tests {
         std::str::FromStr,
     };
 
-    pub fn default_for_chain(chain_id: u64, client: Client) -> Result<UniV3SubgraphClient> {
-        let base_url = Url::parse("https://api.thegraph.com/subgraphs/name/").expect("invalid url");
-        UniV3SubgraphClient::for_chain(&base_url, chain_id, client)
+    pub fn default_for_chain(client: Client) -> Result<UniV3SubgraphClient> {
```

Comment:
```suggestion
    pub fn default(client: Client) -> Result<UniV3SubgraphClient> {
```

This logic is no longer correct. It now uses some nonexistent subgraph name (I'm not sure how these tests are used, but this is likely breaking the integration tests).